--- a/pkg/debian/debian/dirs
+++ b/pkg/debian/debian/dirs
@@ -4,10 +4,6 @@
 # this is where LS_HOME is stored, so let's leave it as is for now
 var/lib/logstash
 
-# Need to create logstash's own directory to track the pid since the daemon is not run as root
-# and doesn't have permission to write the pid to /var/run (which would be the preferred location).
-var/run/logstash
-
 # Store logstash log file in it's own directory since they can become rather large and in the future
 # rotating logs can be easily added.
 var/log/logstash
--- a/pkg/debian/debian/logstash.postinst
+++ b/pkg/debian/debian/logstash.postinst
@@ -46,8 +46,8 @@
         # make sure logstash can delete everything in /var/cache/logstash to
         # re-explode jar.
 
-        chown -R $LS_USER:adm /var/cache/logstash /var/run/logstash
-        chmod -R 750               /var/cache/logstash /var/run/logstash
+        chown -R $LS_USER:adm /var/cache/logstash
+        chmod -R 750               /var/cache/logstash
     ;;
 
     abort-upgrade|abort-remove|abort-deconfigure)
--- a/pkg/debian/debian/logstash.postrm
+++ b/pkg/debian/debian/logstash.postrm
@@ -6,7 +6,7 @@
     purge)
         userdel logstash || true
         rm -rf /var/lib/logstash /var/log/logstash \
-               /var/run/logstash /var/cache/logstash
+               /var/cache/logstash
     ;;
 
     remove|upgrade|failed-upgrade|abort-install|abort-upgrade|disappear)
