--- a/lib/logstash/outputs/s3.rb
+++ b/lib/logstash/outputs/s3.rb
@@ -14,7 +14,7 @@
 
 # S3 plugin allows you to do something complex, let's explain:)
 
-# S3 outputs create temporary files into "/opt/logstash/S3_temp/". If you want, you can change the path at the start of register method.
+# S3 outputs create temporary files into "/usr/share/logstash/S3_temp/". If you want, you can change the path at the start of register method.
 # This files have a special name, for example:
 
 # ls.s3.ip-10-228-27-95.2013-04-18T10.00.tag_hello.part0.txt
@@ -253,7 +253,7 @@
  public
  def register
    require "aws-sdk"
-   @temp_directory = "/opt/logstash/S3_temp/"
+   @temp_directory = "/usr/share/logstash/S3_temp/"
 
    if (@tags.size != 0)
        @tag_path = ""
